app/src/main/res/layout/fragment_add_product_nutrition_facts.xml

```diff
@@ -514,6 +514,7 @@
                 android:gravity="center_vertical"
                 android:hint="@string/nutrition_alcohol"
                 android:inputType="textVisiblePassword"
+                android:imeOptions="actionDone"
```

Comment:
Again we might not be able to go to the additional nutrients from the alcohol field using the soft keyboard​. Can't we use `android:nextFocusDown` instead of action done since it is used for physical keys.

Reply:
@jaindiv26 @huzaifaiftikhar Realistically, you would want `actionNext` on every edittext to ensure that the app drops down to the next available text area, and then `actionDone` on the final one.

Reply:
+1, I agree with @Karljoones :)

Reply:
@teolemon @Karljoones @huzaifaiftikhar I've solved this issue. Check out this screen recording. https://drive.google.com/file/d/1NDfmJxsHMsT0fL1wyv4y6uLSmHJzqgz9/view?usp=sharing

Reply:
@Karljoones @huzaifaiftikhar I've saved the last edit text in a variable and when a new edit text is created last edit text nextFocusDown is set to new one set the IME option of the last edit text to DONE and save the newly created edit text to the variable, you can see the result in the screen recording.

Reply:
@teolemon @Karljoones @huzaifaiftikhar Any updates?